src/amIWilson.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/**
+ * Wilson primes satisfy the following condition:
+ *
+ * 1) Let Num represent a prime number.
+ * 2) Then ((Num-Num)! + 1) / (Num * Num) should give a whole number.
+ *
+ * Implement amIWilson function:
```

Comment:
function names should start with a verb https://mate-academy.github.io/style-guides/javascript.html#naming--camelCase

perhaps isWilsonPrime?